server/api/router.go

```diff
@@ -237,6 +237,7 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	registerFunc(clusterRouter, "/regions/confver", regionsHandler.GetTopConfVerRegions, setMethods("GET"))
 	registerFunc(clusterRouter, "/regions/version", regionsHandler.GetTopVersionRegions, setMethods("GET"))
 	registerFunc(clusterRouter, "/regions/size", regionsHandler.GetTopSizeRegions, setMethods("GET"))
+	registerFunc(clusterRouter, "/regions/topkeys", regionsHandler.GetTopKeysRegions, setMethods("GET"))
```

Comment:
How about
```suggestion
	registerFunc(clusterRouter, "/regions/keys", regionsHandler.GetTopKeysRegions, setMethods("GET"))
```

Reply:
the path of `regions/keys` has been used for key scan.

Reply:
key scan uses `regions/key`

Reply:
fix